### ISSUE TITLE
DPR evaluation scripts

### ISSUE THREAD

**Issue by MXueguang** [castorini/pyserini#369]
Now the DPR evaluation scripts is stored in `tools/scripts/dpr`, 
To evaluate the dpr retrieval result we:
1. get the retrieval result in trec format
2. pull raw texts from anserini index by script `scripts/dpr/convert_trec_run_to_retrieval_json.py`
3. run evaluation script on the generated json file `tools/scripts/dpr/evaluate_retireval.py`

we want to move it within `pyserini` so people can evaluate dpr with just `pip install`

**Comment by lintool** [comment]
We can lump into this issue or create another one:

```
$ python tools/scripts/dpr/evaluate_retrieval.py --retrieval runs/run.nq-test.dpr.bf.bm25.json --topk 20
$ python tools/scripts/dpr/evaluate_retrieval.py --retrieval runs/run.nq-test.dpr.bf.bm25.json --topk 100
Top20  accuracy: 0.8260387811634349
Top100 accuracy: 0.8858725761772853
```

We should allow the ability to specify multiple _k_ values at once, e.g., `--topk 20 100`

**Comment by MXueguang** [comment]
> We should allow the ability to specify multiple k values at once, e.g., --topk 20 100

We support this already, I'll update the doc

**Comment by MXueguang** [comment]
regarding this issue,
how about move `scripts/dpr/convert_trec_run_to_retrieval_json.py` to `pyserini/process`
and move `tools/scripts/dpr/evaluate_retireval.py` to `pyserini/eval`?

**Comment by lintool** [comment]
> how about move `scripts/dpr/convert_trec_run_to_retrieval_json.py` to `pyserini/process`

Isn't this conversion for the purposes of evaluation? In which case, it should be in `eval` also...

**Comment by MXueguang** [comment]
I see, so simply move these two scripts into `pyserini/eval`

**Comment by lintool** [comment]
I think so? Not urgent, though - maybe part of next release after this current one we're working on?

**Comment by MXueguang** [comment]
sure

**Comment by lintool** [comment]
Closed by #397
